src/components/Calculator.js

```diff
@@ -1,40 +1,76 @@
 import React, { Component } from 'react';
+import calculate from '../logic/calculate';
 
 export default class Calculator extends Component {
```

Comment:
- [ ] The Calculator component should import the `calculate.js & operate.js` files from the logic directory currently, you imported `calculate` only please work on that